index.html

```diff
@@ -24,20 +26,104 @@
     </div>
 
     <nav class="nav-bar">
-      <a class="header-logo" href="#"></a>
+      <a class="header-logo" href="index.html"></a>
       <a class="hamburger" href="#"><i id="menu-icon" class="fa-solid fa-bars" onclick="changeIcon(this)"></i></a>
       <a class="menu-open">
-        <ul class="menu-list">
-          <li><a href="#about-page">About</a></li>
-          <li><a href="#">Program</a></li>
-          <li><a href="#">Sponsor</a></li>
-          <li><a href="#">News</a></li>
-          <li><a href="#">Live Score</a></li>
-          <li><a href="#">Participate</a></li>
+        <ul id="menu-list" class="menu-list">
+          <li><a id="list-item1" href="about.html">About</a></li>
+          <li><a id="list-item2" href="#">Program</a></li>
+          <li><a id="list-item3" href="#">Sponsor</a></li>
+          <li><a id="list-item4" href="#">News</a></li>
+          <li><a id="list-item5" href="#">Live Score</a></li>
+          <li><a id="list-item6" href="#">Participate</a></li>
         </ul>
       </a>
```

Comment:
- [ ] [Optional] I would suggest you please stick to the design as much as possible. Please use the same color for the navbar. See the image below for reference
### Your design
![your](https://user-images.githubusercontent.com/81727988/218166642-e41fe1b9-aaa9-4238-803b-c59e0647623d.png)

### Original design
![original](https://user-images.githubusercontent.com/81727988/218166689-f835b77f-797f-4319-82a7-145bd7238952.png)

Reply:
Made changes. Thank you for noticing this design.